app/views/errors.py

```diff
@@ -31,7 +31,7 @@ def add_cache_control(response):
 @errors_blueprint.app_errorhandler(NoTokenException)
 def unauthorized(error=None):
     log_exception(error)
-    return _render_error_page(401)
+    return render_theme_template('default', 'session-expired.html'), 401
```

Comment:
Not a fan of this, so we showing the session-expired page for all 401's. No token/unauthorised we are putting up the message "To help protect your information we have signed you out" and the page title is "Save Successful"